src/nvim/option.c

```diff
@@ -1624,10 +1624,10 @@ static vimoption_T
    {(char_u *)FALSE, (char_u *)0L} SCRIPTID_INIT},
   {"wildmenu",    "wmnu", P_BOOL|P_VI_DEF,
```

Comment:
Do we even want this to be defined as `P_VI_DEF`? I don't see a Vi default documented for them in `options.txt`. Same with `wildmode`.